src/utils/index.js

```diff
@@ -9,14 +10,17 @@ var config      = require( '../config' ),
     log         = require( '../log' ),
     connStatus  = require( '../connection' );
 
+// Placeholder image to show when 404
+var placeholderImage = fs.readFileSync( './placeholders/404.jpg' );
```

Comment:
So this filepath, while I suppose it should work, it heavily depends on the fact that the nodejs server execution command is ran inside of the base project folder. A more dependable way of doing this is to use __dirname, which gives you the current directory that this JS file resides in, and relatively referencing this file.